src/parser/mod.rs

```diff
@@ -8349,6 +8349,13 @@ impl<'a> Parser<'a> {
     pub fn parse_object_name(&mut self, in_table_clause: bool) -> Result<ObjectName, ParserError> {
         let mut idents = vec![];
         loop {
+            if self.dialect.supports_object_name_double_dot_notation()
+                && !idents.is_empty()
```

Comment:
I'm debating if I should force it only for missing schema. Which means only accepts `x..y` and rejects stuff like `x.y..z`. Forcing this seems like a semantic validation that we are trying to avoid.